### DISCUSSION TITLE
How can I make <space> be part of match?

### DISCUSSION THREAD

**Discussion by ittayd** [marlonrichert/zsh-autocomplete#665] [Q&A]
When I go into history search using ctrl-r I want the space char to be part of the search string. Right now it terminates the search. I want search termination to be either tab, enter or esc. How can I achieve that?

**Comment by marlonrichert** [comment]
That's now it works for me. For example, when I press `^R` and type `git `, then the trailing space is part of the search and the search is not aborted. Have you tried doing so with the latest version of Autocomplete?
